src/main/java/ch/njol/skript/expressions/ExprTool.java

```diff
@@ -42,75 +30,120 @@
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.registrations.Classes;
 import ch.njol.skript.util.Getter;
+import ch.njol.skript.util.slot.CursorSlot;
 import ch.njol.skript.util.slot.EquipmentSlot;
 import ch.njol.skript.util.slot.InventorySlot;
 import ch.njol.skript.util.slot.Slot;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
-@Name("Tool")
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.inventory.ClickType;
+import org.bukkit.event.inventory.InventoryClickEvent;
+import org.bukkit.event.player.PlayerBucketEmptyEvent;
+import org.bukkit.event.player.PlayerBucketEvent;
+import org.bukkit.event.player.PlayerBucketFillEvent;
+import org.bukkit.event.player.PlayerItemHeldEvent;
+import org.bukkit.inventory.EntityEquipment;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.PlayerInventory;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Tool/Offhand")
 @Description("The item an entity is holding in their main or off hand.")
-@Examples({"player's tool is a pickaxe",
+@Examples({
+	"player's tool is a pickaxe",
 	"player's off hand tool is a shield",
 	"set tool of all players to a diamond sword",
-	"set offhand tool of target entity to a bow"})
-@Since("1.0")
+	"set offhand tool of target entity to a bow"
+})
+@Since("1.0, 2.2-dev37 (offhand)")
 public class ExprTool extends PropertyExpression<LivingEntity, Slot> {
+
 	static {
 		Skript.registerExpression(ExprTool.class, Slot.class, ExpressionType.PROPERTY,
-			"[the] ((tool|held item|weapon)|1¦(off[ ]hand (tool|item))) [of %livingentities%]",
-			"%livingentities%'[s] ((tool|held item|weapon)|1¦(off[ ]hand (tool|item)))");
+				"[the] ((tool|held item|weapon)|offhand:(off[ ]hand (tool|item))) [of %livingentities%]",
+				"%livingentities%'[s] ((tool|held item|weapon)|1¦(off[ ]hand (tool|item)))"
+		);
```

Comment:
Forgot to update 2nd pattern's offhand pattern.
Also it might be easier to clean these patterns up vs. keeping nearly zero changes

I initially thought about this but it enables `item of ...` which conflictis with `ExprItemFrameSlot`
```
[offhand:off[ ]hand] (tool|weapon|[held] item) [of %livingentities%]
```